backend/pkg/kubeconfig/kubeconfig.go

```diff
@@ -281,334 +304,412 @@ func LoadContextsFromBase64String(kubeConfig string, source int) ([]Context, []e
 		return nil, []error{fmt.Errorf("error decoding base64 kubeconfig: %v", err)}
 	}
 
-	return loadContextsFromData(kubeConfigByte, source)
+	skipProxySetup := source != KubeConfig
+
+	return loadContextsFromData(kubeConfigByte, source, skipProxySetup)
 }
 
-// loadContextsFromData is a helper function that loads contexts from a byte slice.
-// It parses the byte slice as a YAML file and validates the parsed data.
-// It then creates contexts from the parsed data and returns them.
-// It returns any errors that occurred during parsing or validation.
-func loadContextsFromData(data []byte, source int) ([]Context, []error) {
+// loadContextsFromData loads contexts from a kubeconfig data.
+// It unmarshals the kubeconfig data, extracts the contexts, and processes each context.
+// It returns the processed contexts and any errors that occurred during the process.
```

Comment:
Can you please add some documentation on how to discover if a context is valid, and that it returns all contexts even if they are invalid?

It looks to me there's a bug, because errors are only conditionally added. So there could be 3 contexts and 2 errors. But are the errors for the 1,2, or 3rd contexts? There's no way to tell.

Reply:
We are now not checking which context it valid or not, we are now just returning all contexts and errors